app/Scripts/Player.gd

```diff
@@ -37,6 +37,10 @@ func _ready():
 		
 	self.set_meta("Player", true)
 	
+func add_damage(value: int) -> void:
+	PlayerStats.Health -= value
+
+	
```

Comment:
damage is applied by hitbox of other entities. This way different objects can apply different damages.